test/script.clj

```diff
@@ -34,6 +34,79 @@
        (with-out-str (aws/doc s3 :ListBuckets))
        "Returns a list of all buckets")))
 
+(require '[pod.babashka.aws.credentials :as creds])
+
+(defmacro with-system-properties [props & body]
+  `(let [props# (System/getProperties)]
+     (try
+       (doseq [[k# v#] ~props]
+         (System/setProperty k# v#))
+       ~@body
+       (finally
+         (System/setProperties props#)))))
+
+(defn create-temp-dir [prefix affix]
+  (let [f (java.io.File/createTempFile prefix affix)]
+    (.delete f)
+    (.mkdir f)
+    f))
+
+
+(defn create-aws-credentials-file [content]
+  (let [temp-dir (create-temp-dir "pod-babashka-aws-" "-credentials")
+        creds-file (clojure.java.io/file temp-dir ".aws/credentials")]
+    (.mkdirs (.getParentFile creds-file))
+    (spit creds-file content)
+    ;; Return home dir
+    (.getPath temp-dir)))
+
+(deftest aws-credentials-test
```

Comment:
I link to the test/script.clj script from the README as an example how to use this pod.
Since these are implementation detail tests, maybe we can pull them out in a separate script?

Reply:
Yeah good point. And maybe a seperate file if still too big

Reply:
Fixed in efd14afc1e9e82a9875ee08e0d99fd6e98c1d425

Run as `bb --clojure -M test/credentials_test.clj`

Where do we aggregate the running of these tests? Not sure who to make this part of script.clj

Reply:
I can hook it up in a test script after merge.